bh_core.sublime-settings

```diff
@@ -446,6 +446,17 @@
             "language_list": ["LaTeX", "LaTeX (TikZ)", "knitr (Rnw)"],
             "sub_bracket_search": "true",
             "enabled": true
+        },
+        // Pascal
+        {
+            "name": "pascal",
+            "open": "(?<=[\\s;])(try|class|record|begin|unit|repeat)\\b",
```

Comment:
I've updated the RegEx patterns with `interface` and to requre `= ` before certain keywords that are reused.  I've also added a `bh_module` to make sure the various mates match up properly.